yarn.lock

```diff
@@ -11603,10 +11604,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"preact@npm:^10.5.13":
-  version: 10.13.2
-  resolution: "preact@npm:10.13.2"
-  checksum: 3bc98aa09bcd297eb59abd7e4f3a4d499b8e345bd68b922f7678ef105ba4721dc9b9940b221e6e3443f957d51402fe407bb96ccaa0a4b65d6808ca8a3be76bfa
+"preact@npm:^10.16.0, preact@npm:^10.5.13":
+  version: 10.16.0
+  resolution: "preact@npm:10.16.0"
+  checksum: 47a91f47d583b68a4afe971a7f992c06547df6d637cadf56eb3b69fee1fb202659b199af37d0e1a90637385144cadd75aa40acdb4e125cc4b3155e2883c24c07
```

Comment:
@Murderlon it does feel a bit weird to have to update Prettier as part of a Uppy update 🤔

Reply:
You mean preact? Not sure that's weird, it's part of the UI?

Reply:
The weird thing is we have dependencies on two different versions of Preact, which would likely cause problems, wdyt?

Reply:
every UI plugin has it's own Preact dependency, which makes sense, but we should probably hard pin it to the same version everywhere.

Reply:
Yes, and now `@uppy/utils` has its own Preact dependency that is inconsistent with the UI plugins.